src/containers/sb3-downloader.jsx

```
@@ -7,6 +7,9 @@ import downloadBlob from '../lib/download-blob';
 import {setProjectUnchanged} from '../reducers/project-changed';
 import {showStandardAlert, showAlertWithTimeout} from '../reducers/alerts';
 import {setFileHandle} from '../reducers/tw';
+import {loadRecentProjects} from '../reducers/recent-projects';
+import {loadRecentProjects} from '../reducers/recent-projects';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'loadRecentProjects'. Consider to delete it or rename it.

'loadRecentProjects' is defined here:


(lint/suspicious/noRedeclare)